source/console/README.md

```diff
@@ -0,0 +1,26 @@
+# Windows Console Renderer for Adaptive Cards
```

Comment:
I would suggest making moving it into a HACK or experimental folder to make sure that people aren't confused.